articles/cloud-storage/cs-how-use-file-browsers.md

```diff
@@ -146,78 +146,64 @@ To install and configure S3 Browser:
+This section takes you through installing and configuring browser access that enables users to browse the contents of Cloud Storage using their web browser.
 
-Atmos extensions are available for both Chrome and Firefox. Although the extensions included in this article were natively built for the Atmos platform, they also work with ECS using Atmos endpoints and therefore work well with Cloud Storage.
+An EMC created HTML5 based application can be installed to your Atmos bucket, which generates a URL that enables customers to share with users who require access to cloud storage.
 
-S3 extensions are also available for both browsers, but none of them include the functionality to customise the server host name, which is a requirement for connecting to Cloud Storage. If we find S3 browser extensions that offer this functionality in the future, we'll update this article.
+   > [!NOTE]
+   > If you have multiple buckets under the same endpoint these can all be accessed from a single URL, however if you have multiple buckets hosted in both Cloud Storage regions, you will need to install the application on both regions and access the relevent buckets using the relevent URL.
 
-### Chrome — Atmos for Chrome
+S3 extensions are also available for browsers, but none of them include the functionality to customise the server host name, which is a requirement for connecting to Cloud Storage. If we find S3 browser extensions that offer this functionality in the future, we'll update this article.
 
-To install and configure Atmos for Chrome:
+### Atmos HTML5 Application
 
-1. Download and install the [Atmos for Chrome extension](https://chrome.google.com/webstore/detail/atmos-for-chrome/befofpjhdclgccimklkaheifjajbcmnk).
+   > [!NOTE]
+   > For installation and initial configuration of the Atmos HTML5 application, you will need the following Prerequisite information:
+   > Atmos Access Key - in the form of SubtennantID/User
+   > Atmos Secret Key - Shared Secret Key
+   > A folder within an existing bucket or new bucket to host the GUI files e.g. /<bucket_name>/HTML5UI or /HTML5UI
 
-2. After you complete the installation, you'll see a globe icon in your browser toolbar. Click this icon to launch the extension.
+To install and configure the Atmos HTML5 application on windows:
 
-    ![Atmos for Chrome toolbar icon](images/cs-file-browsers-ext-1.png)
+1. Download and extract the [Atmos HTML5 application](INSERT URL HERE).
 
-3. To connect the extension to Cloud Storage, click the **Manage Accounts** button.
+2. Open a command prompt and navigate to the extracted files and run the following command: upload.bat WebContent\main.html /<bucket_name>/<optional_path>/
 
-4. Add an account for Cloud Storage using your credentials, which you can find in the *Storage* section of the UKCloud Portal.
+3. The installer script will run and prompt for the following information.
```

Comment:
I think you can just reduce this to:

The installer script will prompt for the following information:

(Put a colon at the end instead of a full stop)